Bio/PDB/ccealignmodule.c

```diff
@@ -211,6 +211,149 @@ calcS(
     return S;
 }
 
+static const double tableZtoP[] = {
```

Comment:
Could I trouble you to leave a comment here explaining what P and Z are? I assume Z is just a Z-score, but what's P, a probability?

Reply:
Yes, that's correct. I'll add the comments.